packages/template-ui/package.json

```diff
@@ -20,7 +20,8 @@
     "vue": "^2.6.11",
     "vue-clamp": "^0.3.1",
     "vue-router": "^3.2.0",
-    "vuex": "^3.5.1"
+    "vuex": "^3.5.1",
+    "vue-material-design-icons": "^4.12.1"
```

Comment:
I think this is because these icons are better than the default ones that we've in the bootstrap-vue lib: https://bootstrap-vue.org/docs/icons

Reply:
Yes, currently we have a mix of icons, some from Bootstrap (provided by bootstrap-vue) and some for Material. Joana started adding Material icons for the play buttons, then in more and more places. I think we should use only the Material ones. I just created https://phabricator.endlessm.com/T32939 for unifying this. Needs Joana's decision first.